### PR TITLE
Add object list in config

### PR DESCRIPTION
**Pull request by EPashkin**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,21 +1,54 @@
...
+Usage: gir [-d <girs_dir>] [<gir>]
+
+Options:
+    -d PATH            Directory for girs
 ";
 
 fn main() {
     let args = Docopt::new(USAGE).unwrap()
         .parse().unwrap_or_else(|e| e.exit());
+    //println!("{:?}", args);
```
Comment (gkoz) [inline src/main.rs]: Do these debugging lines still serve any purpose?

File: src/main.rs [hunk 2]
```diff
@@ -1,21 +1,54 @@
...
+    //println!("Dir: {}", args.get_str("-d"));
+    //println!("File: {}", args.get_str("<gir>"));
+    
+    let toml = read_toml("Gir.toml");
+    // println!("Toml: {:?}", toml);
+    
+    let girs_dir = match args.get_str("-d") {
+        "" => toml.lookup("options.girs_dir")
```
Comment (gkoz) [inline src/main.rs]: I still believe this needs a more specific name. What's a "gir"? `gir = "Gtk-3.0"`? Could be `library`, `namespace`, something with a meaning.

File: src/main.rs [hunk 3]
```diff
@@ -1,21 +1,54 @@
...
+                  .unwrap_or_else(|| panic!("No options.girs_dir in config"))
+                  .as_str().unwrap(),
+        a => a
+    };
+    //println!("Dir: {:?}", girs_dir);
+
+    let gir = match args.get_str("<gir>") {
+        "" => toml.lookup("options.gir")
```
Comment (gkoz) [inline src/main.rs]: I still believe this needs a more specific name. What's a "gir"? gir = "Gtk-3.0"? Could be library, namespace, something with a meaning.

Comment (EPashkin) [inline src/main.rs]: Parameter `<library>` with `let library_name = match ...` ?

Comment (gkoz) [inline src/main.rs]: Seems okay.

**Comment by EPashkin** [comment]
Removed all debug prints, except 'objects'

**Comment by gkoz** [comment]
Thanks.

Consider that the generator is capable of knowing if something is a `GObject`, a `GtkWidget`, etc so putting that into the config seems like wasted effort.

**Comment by EPashkin** [comment]
Yes. Don't looked at `library` at moment of writing.
I fix this and name (to "Gtk.Widget") in next PR.

**Pull request merged**